### PR TITLE
Adding "Start missions in Chase View by default" flag in game_settings.tbl

### LINKED ISSUE #3152: FR: set third person "chase" camera as default
Possible flag to set in game_settings.tbl. Could be useful for arcadey TCs like Starfox: Event Horizon and Solaris. The option to switch between cockpit "first person" and chase "third person" views is still there but the game defaults to the chase view.

### PR DESCRIPTION
**Pull request by Strygon**
If the mod has "$Player starts in Third Person View by Default: TRUE" in their game_settings.tbl or any such .tbm, missions will automatically put the player in the third person chase view. This was requested as a FR by Darius on January 17th.

This will NOT stop the player from going back into first person by pressing the key for it.

closes #3152

### CODE REVIEW

File: code/mod_table/mod_table.cpp [hunk 1]
```diff
@@ -62,6 +62,7 @@ bool Use_engine_wash_intensity;
 bool Framerate_independent_turning; // an in-depth explanation how this flag is supposed to work can be found in #2740 PR description
 bool Ai_respect_tabled_turntime_rotdamp;
 bool Swarmers_lead_targets;
+bool Third_Person_is_Default;
```
Comment (BMagnu) [inline code/mod_table/mod_table.cpp:65]: This should be `Chase_view_default `as well.

File: code/mod_table/mod_table.cpp [hunk 2]
```diff
@@ -654,6 +659,7 @@ void mod_table_reset()
 	Use_engine_wash_intensity = false;
 	Framerate_independent_turning = true;
 	Ai_respect_tabled_turntime_rotdamp = false;
+	Third_Person_is_Default = false;
```
Comment (BMagnu) [inline code/mod_table/mod_table.cpp:662]: This should also be `Chase_view_default`

**Review by BMagnu — CHANGES_REQUESTED** [review]
Apart from the few bad bools it seems good

**Review by BMagnu — APPROVED** [review]
LGTM

**Review by Baezon — APPROVED** [review]

**Review by Goober5000 — CHANGES_REQUESTED** [review]
This should be a missions flag, not a game_settings flag.

File: code/playerman/playercontrol.cpp [hunk 3]
```diff
@@ -1379,7 +1379,15 @@ void player_level_init()
 	Viewer_external_info.preferred_distance = 0.0f;
 	Viewer_external_info.current_distance = 0.0f;
 
-	Viewer_mode = 0;
+	
+	if (Chase_view_default == true)
```
Comment (Goober5000) [inline code/playerman/playercontrol.cpp:1383]: This is a redundant boolean comparison, which some compilers will complain about.

**Review by Goober5000 — CHANGES_REQUESTED** [review]

**Comment by Baezon** [comment]
I don't see why it should be a mission flag, all the current examples of it are as a stylistic choice for the entire mod. While it's not inconceivable that someone might want it per mission, its clear the vastly more common scenario is for the entire mod.

**Comment by Goober5000** [comment]
It really should be a per-mission flag.  If we want to use game_settings to make the mission flag default to true for every mission, that's fine, but this setting should be available on a per-mission basis.  I can already think of one use case and I'm sure there are a few I haven't thought of.

**Review by Goober5000 — APPROVED** [review]
Per discussion with @Baezon I'll approve this, and I'll add the mission flag in a separate PR.

**Pull request merged**